### ISSUE TITLE
TW stopped displaying old todolists after upgrading TW-Todolist plugin to newest version

### ISSUE THREAD

**Issue by s-kaczmarek** [kookma/TW-Todolist#6]
Hi,

After updating TW-Todolist plugin to newest versions all of my todolists disappeared from the wiki [BobLinux]. All filest starting with **$__plugins_kookma_todolist** are still present in the filesystem, but are just not displaying lists in the wiki.

Any idea how to restore them? 

If you need more information please tell me which one do you need.

**Comment by kookma** [comment]
@s-kaczmarek 
I hope you had a backup of your Tiddlywiki. By the way in the tutorial there is a section on upgrading, see

https://kookma.github.io/TW-Todolist/#Update%20from%20Todolist%201.0.0

Todilist 1.2 uses a slightly different naming rules for state tiddlers. Simply follow the procedure! Please keep another backup before you make sure everything goes well.

**Comment by s-kaczmarek** [comment]
Yeah, I have backup - I keep my entire TW in git repo, so I can roll back to previous commits when it was ok ;) It looks like easy fix. I will try it and let you know. Many thanks!
